internal/host/hostcommands/instruction_manager.go

```diff
@@ -151,6 +165,19 @@ func (i *InstructionManager) GetNextSteps(ctx context.Context, host *models.Host
 	return returnSteps, nil
 }
 
+func generateDisabledStepsMap(log logrus.FieldLogger, disabledSteps []models.StepType) map[models.StepType]bool {
+	result := map[models.StepType]bool{}
```

Comment:
make will be clearer

Reply:
true, will be addressed in https://github.com/openshift/assisted-service/pull/2276